Update Ogi.tsx for proper naming convention.

kalittles opened the pull request:
Minor update to displayed text for using the proper number of Ogi Namikiris.

kalittles committed:
Update Ogi.tsx for proper naming convention.

Minor update to displayed text for using the proper number of Ogi Namikiris.

kalittles committed:
Ran yarn extract to fix the xiva.extract.failure message extraction test failure.

kalittles committed:
Ran yarn extract to fix the xiva.extract.failure message extraction test.

Had to finish the entire yarn extract.

kalittles commented:
Accidently hit commit before the YARN extract finished, that's why the diff looks super weird. Should be fine now.

kalittles committed:
Merge branch 'xivanalysis:endwalker' into endwalker

AzariahAeroborn reviewed (changes requested):
Hey, thanks for the commit! We require a change log entry in the `/parser/<JOB>/index.ts` file for any updates / bug fixes that change job logic, and you'll need to upload an avatar to /src/data/avatar and add yourself to the /src/data/CONTRIBUTORS.ts file. If you have any questions, please feel free to drop by our Discord and tag yourself with the Developer role.

kalittles committed:
Adding requests changes for contributor information.

I added an avatar and updated contributors as requested in the comment on PR 1554

kalittles commented:
Thank you for the response!

I've made the requested changes, although I feel like I should definitely do some more in the future if being listed as a contributor is a requirement, haha. I just look at my XIVAnalysis a lot and kept seeing it, so I wanted to change it. I feel a little disingenuous being listed when all I did was fix a typo.

No actual logic was changed, so I didn't update /parser/SAMURAI/index.ts if that's okay.

kalittles committed:
Making review changes requested for contributions section.

kalittles committed:
Fixing lint test failure - comma dangle.

AzariahAeroborn commented on src/parser/jobs/sam/index.js:
```diff
@@ -26,6 +26,8 @@ export const SAMURAI = new Meta({
 
 	contributors: [
 		{user: CONTRIBUTORS.RYAN, role: ROLES.DEVELOPER},
+		{user: CONTRIBUTORS.ACKWELL},
```
This looks like an unintentional copy paste error.
```suggestion
```

kalittles replied:
Okay, I've made the changes.

And no problem! No need to apologize. I'm glad something else useful came from it! I appreciate you walking me through the process, and hopefully I can contribute in more meaningful ways in the future.

AzariahAeroborn reviewed (changes requested):
Sorry for not being more specific with my original feedback, I need to update my template... if you would, please shrink the avatar to 100x100 px or smaller, preferably under 100KB in size.

In hindsight, I do agree that this small copy change is ok without a change log.

kalittles committed:
Fixing contributor copy paste problem.

AzariahAeroborn committed:
Merge branch 'endwalker' into endwalker

AzariahAeroborn committed:
fix linting problems from the web merge editor
